core/.env.test

```diff
@@ -1,5 +1,6 @@
 DATABASE_URL=postgresql://postgres:postgres@localhost:5433/postgres
 PUBPUB_URL=http://localhost:3000
+NEXT_PUBLIC_PUBPUB_URL=http://localhost:3000
```

Comment:
was this to debug e2e failure maybe? afaik we're not using `NEXT_PUBLIC` env vars anymore

Reply:
that also seems to be fine

Reply:
Yeah I added it because I'm using it, but your way sounds better so I'll fix it! Maybe this is what's making the e2e tests break? Also fwiw this change was an error, since .env.test isn't actually used when we run the playwright tests (only the vitest tests).

Reply:
Actually this causes the build to fail. Did you have a way around that?

Reply:
yes, 

```
	baseUrl: typeof window === "undefined" ? env.PUBPUB_URL : window.location.origin,
```